Add note about TEM (#1136)
UniversalRobots/Universal_Robots_ROS2_Driver: ur_moveit_config/doc/index.rst

Now you should be able to use the MoveIt Plugin in rviz2 to plan and execute trajectories with the
robot as explained `here <https://moveit.picknik.ai/main/doc/tutorials/quickstart_in_rviz/quickstart_in_rviz_tutorial.html>`_.

.. note::
   Usually, MoveIt uses trajectory execution monitoring (TEM). If executing a trajectory takes too long,
   e.g. because the action server died or the robot is blocked, this will lead to an error in

>    trajectory execution. However, with the scaled joint trajectory controller used by this driver,

Reviewer: ```suggestion
   trajectory execution. However, with the scaled joint trajectory controller offered by this driver,
```

>    this doesn't make much sense (as long as TEM is not aware of the scaling), as execution time can

Reviewer: Perhaps "does not work" or "is incompatible with".

It still makes sense, it's just that with the current implementation on MoveIt's side it will result in undesirable behaviour, as it always sees execution time violations.

Author: I did not want to place TEM as a bad idea or anything, so "doesn't make sense" isn't the best wording, yes. "does not work" also isn't entirely true, as it does work in terms that is monitors whether the trajectory is executed in the expected time. Howabout "isn't necessarily compatible with"? I mean, it is pretty compatible with it, as long as no scaling is happening.

>    Usually, MoveIt uses trajectory execution monitoring (TEM). If executing a trajectory takes too long,
>    e.g. because the action server died or the robot is blocked, this will lead to an error in
>    trajectory execution. However, this isn't necessarily compatible with the scaled joint
>    trajectory controller (as long as TEM is not aware of the scaling), as execution time can
>    be unbounded on purpose. Hence, TEM is disabled in the example MoveIt configuration:

Reviewer: Another suggestion:

```suggestion
The MoveIt configuration provided here has Trajectory Execution Monitoring *disabled*, as the Scaled Joint Trajectory Controller may cause trajectories to be executed at a lower velocity than they were originally planned by MoveIt. MoveIt's TEM however is not aware of this deliberate slow-down due to scaling and will in most cases unnecessarily (and unexpectedly) abort goals.

Until this incompatibility is resolved, the default value for `execution_duration_monitoring` is set to `false`. Users who wish to temporarily (re)enable TEM at runtime (for use with other, non-scaling controllers) can do so using the ROS 2 parameter services supported by MoveIt.
```

a bit more text, but perhaps more specific about what's going on, what would happen if it wasn't disabled, which parameter is specifically responsible for disabling it and how users could (temporarily perhaps) re-enable it at runtime.

Author: That sounds good, thank you!

>    be unbounded on purpose. Hence, TEM is disabled in the example moveit configuration:

Reviewer: ```suggestion
   be unbounded on purpose. Hence, TEM is disabled in the example MoveIt configuration:
```
